docs/blog/2019-01-30-why-themes/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Why Themes?"
+date: "2019-01-30"
+author: "Kyle Mathews"
+tags: ["themes"]
+---
+
+> _A site as a function of your data_
+
+As some of you might have heard (or hoped), we're working on adding theme support to Gatsby. This blog posts is a short intro into how we're thinking about themes in Gatsby.
+
+This post is short on examples and concrete details. See [our other theme blog posts by Chris Biscardi](/blog/tags/themes) and his recent [talk at Gatsby Days on themes](https://youtu.be/wX84vXBpMR8).
+
+## Introduction & Goal
+
+Themes is a powerful new abstraction under development for Gatsby. They allow a developer to package up a site as a standalone product for others to use.
+
+Abstractions are why programming works. Abstractions allow a developer to express their intent with ease.
+
+For example, every developer needs to write files to a disk — so the hardware and software industry has responded by spending an incredible amount of money and time to make writing data to disk is easy, fast, and reliable.
+
+Developers say Gatsby is "easy to use". Why? Because we’ve put 1000s of hours towards designing, creating, and documenting thoughtful abstractions with our APIs and plugin system. People have responded by creating [100s of new plugins over the last 1.5 years](/plugins/).
+
+Good systems for creating abstractions help people collaborate on shared solutions to common problems. A community can pool together their combined understanding to create a better solution than any one person would make. So with Gatsby, when someone wants to [add google analytics](/packages/gatsby-plugin-google-analytics/?=google), [pull data from a CMS](/packages/gatsby-source-contentful/?=contentful), [optimize their images](/packages/gatsby-image/?=gatsby-image), etc. — instead of researching and experimenting to implement their own solution — they just install a plugin worked on and used by 10s of thousands of fellow web developers.
```

Comment:
What's the `?=google` on these URLs?

Reply:
it's the search term he used to get the link from the plugins page. could be removed

Reply:
I think it generally makes the page better as it filters down the plugin page. Very inconsequential though either way.